### PR TITLE
Added some files

### PR DESCRIPTION
**Pull request by Ahuvim**
added reviewers getByID
added reviewers getAll
added reviewers getByIDHandle
added reviewers getAllHandle

### CODE REVIEW

**Comment by dormesica** [comment]
What is the purpose of `packages/server/src/ddjdj.js`?

**Comment by Ahuvim** [comment]
It is a mistake. Please ignore

File: packages/server/src/api/reviewers/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const getAll = require('./getAll');
+const getByID = require('./getByID');
+
+module.exports = [].concat(getAll,getByID);
```
Comment (dormesica) [inline packages/server/src/api/reviewers/index.js]: Wouldn't it be simpler to write
```javascript
module.exports = [
    getAll,
    getById,
];
```

File: packages/server/src/handlers/reviewers/getByIDHandle.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const interviewers = require('../../DAL/DB');
+
+function getByID(request) {
+
```
Comment (dormesica) [inline packages/server/src/handlers/reviewers/getByIDHandle.js]: Please remove this line

File: packages/server/src/index.js [hunk 3]
```diff
@@ -16,6 +16,8 @@ const server = Hapi.server({
 });
 
 server.route(require('./api/isAlive'));
+server.route(require('./api/reviewers'));
+
```
Comment (dormesica) [inline packages/server/src/index.js]: leave 1 space line.

**Pull request closed without merging**